src/hooks/useServerApi.ts

```diff
@@ -97,16 +98,26 @@ export const useServerApi = () => {
     authRequired = false,
     impersonationData,
     version,
-  }: Omit<RequestOptions, 'additionalHeaders'> & {
+    additionalHeaders,
+  }: RequestOptions & {
     impersonationData?: ImpersonationData;
     version?: string;
   }): Promise<T> => {
     const headers: Record<string, string> = {
-      'x-locale': lang,
+      'x-locale': locale,
       'tenant-key': tenantConfig?.id || '',
       'X-SESSION-ID': await getSessionId(),
+      ...(additionalHeaders ? additionalHeaders : {}),
     };
 
+    if (version) {
+      headers['x-accept-versions'] = version ? version : '1.0.3';
+    }
```

Comment:
Populate the header only when version is explicitly provided.

Reply:
For future use of versioning, we could use a env var. Take note of this as a comment for later.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2455/commits/790d1b1be21f728318ee69be113144597da4c3d0